### ISSUE TITLE
[🐛 Bug]: Test environment jest-environment-selenium cannot be found.

### ISSUE THREAD

**Issue by ldynia** [SeleniumHQ/selenium-ide#1363]
### What happened?

I run my selenium tests from shell -everything works as it should! However, if I execute the same command from a shell script `test.sh` I'll get below error. What causes it and how can I fixt it ?

```bash
$ selenium-side-runner --server http://chromedriver:4444/wd/hub --capabilities "browserName=chrome" /app/tests/proxy.lucky_app.side
info:    Running /app/tests/proxy.lucky_app.side
 PASS  ./DefaultSuite.test.js
  Default Suite
    ✓ test frontend (1282ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        3.329s
Ran all test suites.
```

**tests.sh**
```bash
#!/bin/bash

selenium-side-runner --server http://chromedriver:4444/wd/hub --capabilities "browserName=chrome" /app/tests/proxy.lucky_app.side
```

**Error**

```shell
$ /app/scripts/test.sh
info:    Running /app/tests/proxy.lucky_app.side
● Validation Error:

  Test environment jest-environment-selenium cannot be found. Make sure the testEnvironment configuration option points to an existing node module.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html


### How can we reproduce the issue?

```shell
Run any successful selenium test with selenium-side-runner in terminal. Then put this command into `test.sh` script and run the scrip in a docker container created from `node:17.8-slim` image.

package.json
{
  "dependencies": {
    "fs": "^0.0.1-security",
    "selenium": "^2.20.0",
    "selenium-side-runner": "^3.17.0",
    "selenium-webdriver": "^4.1.1"
  }
}
```

### Relevant log output

```shell
info:    Running /app/tests/proxy.lucky_app.side
● Validation Error:

  Test environment jest-environment-selenium cannot be found. Make sure the testEnvironment configuration option points to an existing node module.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

### Operating System

Ubuntu

### Selenium version

2.20.0

### What are the browser(s) and version(s) where you see this issue?

Headless/Chrome

### What are the browser driver(s) and version(s) where you see this issue?

selenium/standalone-chrome

**Comment by toddtarsi** [comment]
@ldynia - Yeah, thanks for reporting! We're planning to do an alpha release of v4 sometime this month, and I've completely dropped our use of jest-environment-selenium, because just like you, I don't know what it does either honestly haha.

No in all fairness, it actually looks really cool, it's just that I'm we're sharing the exact same runner between local playback and remote playback now, so jest specific assertions are not as valuable to us anymore.

**Comment by ldynia** [comment]
@toddtarsi Thanks for fast response! The reason for my error and this flaky behavior was explicit reference to the latest release of `selenium-webdriver` - at the moment of writing this answer, it's `selenium-webdriver@4.1.1`

**package.json** - error
```json
{
	"dependencies": {
		"fs": "^0.0.1-security",
		"jest-environment-selenium": "^2.1.2",
		"selenium-side-runner": "^3.5.4",
		"selenium": "^2.20.0",
		"selenium-webdriver": "^4.1.1"
	}
}
```

Fix, was to remove `selenium-webdriver@4.1.1` from `package.json`.

**package.json** - no errors

```json
{
	"dependencies": {
		"fs": "^0.0.1-security",
		"jest-environment-selenium": "^2.1.2",
		"selenium-side-runner": "^3.5.4",
		"selenium": "^2.20.0",
	}
}
```
